### DISCUSSION TITLE
minecraft

### DISCUSSION THREAD

**Discussion by Jakobboy** [share121/inter-knot#669] [其他]
![2024-05-19_11 44 58](https://github.com/user-attachments/assets/d351dfa4-f5db-4c84-9a67-d2b0f2a02f48)

分享下minecraft截图

**Comment by Ailian-Qiao** [comment]
![Desktop Screenshot 2024 08 11 - 01 21 50 70](https://github.com/user-attachments/assets/0464d795-d5f4-446f-a7f3-c9fbe05dae1a)
偷我电脑壁纸？

**Comment by shilijia-siweikongjian** [comment]
it3好评

**Comment by cpu644** [comment]
@Ailian-Qiao 你2077快捷方式寄了

**Reply by su2569** [reply]
> @Ailian-Qiao 你2077快捷方式寄了

确实

**Comment by su2569** [comment]
.

**Comment by CrazyFeng114** [comment]
这沓亇是mc？

**Comment by RETRN1** [comment]
是新空洞吗😨😨😨
